h2o-py/h2o/model/PermutationFeatureImportance.py

```diff
@@ -0,0 +1,31 @@
+# -*- encoding: utf-8 -*-
+"""
+H2O Permutation Feature Importance.
+
+:copyright: (c) 2020 H2O.ai
+:license:   Apache License Version 2.0 (see LICENSE for details)
+"""
+
+from h2o.frame import H2OFrame
+from h2o.expr import ExprNode
+
+
+def permutation_varimp(validation_frame, model, use_pandas=True):
+
+    m_frame = H2OFrame._expr(ExprNode("Perm_Feature_importance", validation_frame, model))
+
+    if type(m_frame) is not H2OFrame:
```

Comment:
You should test input frame not your output.
```suggestion
    if type(validation_frame) is not H2OFrame:
```